frontend/src/components/explorer/gemBrowser/Metabolite.vue

```diff
@@ -184,4 +186,10 @@ export default {
 };
 </script>
 
-<style lang="scss"></style>
+<style lang="scss">
+#chebi-img {
+   max-width: 400px;
+   width:100%;
```

Comment:
I couldn't find a scenario where these 2 rules would be required.

```suggestion
```

Reply:
Nice, me neither tbh. Think it was added to avoid the table taking up too much space but as you say it seems to work fine without it